master/internal/config/config_test.go

```diff
@@ -891,90 +891,53 @@ resource_manager:
 }
 
 func TestPickVariation(t *testing.T) {
+	userConfig := MediaAssetVariations{
+		LightHorizontal: "light-horizontal",
+		LightVeritical:  "light-vertical",
+		DarkHorizontal:  "dark-horizontal",
+		DarkVeritical:   "dark-vertical",
+	}
 	tests := []struct {
-		name        string
-		variations  MediaAssetVariations
 		mode        string
 		orientation string
 		expected    string
 	}{
 		{
-			name: "Light Horizontal prioritized",
-			variations: MediaAssetVariations{
-				LightHorizontal: "light-horizontal",
-				LightVeritical:  "light-vertical",
-				DarkHorizontal:  "dark-horizontal",
-				DarkVeritical:   "dark-vertical",
-			},
 			mode:        "",
 			orientation: "",
 			expected:    "light-horizontal",
 		},
 		{
-			name: "Light Vertical when Light Horizontal is empty",
-			variations: MediaAssetVariations{
-				LightHorizontal: "",
-				LightVeritical:  "light-vertical",
-				DarkHorizontal:  "dark-horizontal",
-				DarkVeritical:   "dark-vertical",
-			},
+			mode:        "",
+			orientation: "horizontal",
+			expected:    "light-horizontal",
+		},
+		{
 			mode:        "",
 			orientation: "vertical",
 			expected:    "light-vertical",
 		},
 		{
-			name: "Dark Horizontal when mode is dark",
-			variations: MediaAssetVariations{
-				LightHorizontal: "light-horizontal",
-				LightVeritical:  "light-vertical",
-				DarkHorizontal:  "dark-horizontal",
-				DarkVeritical:   "dark-vertical",
-			},
-			mode:        "dark",
+			mode:        "light",
 			orientation: "",
-			expected:    "dark-horizontal",
+			expected:    "light-horizontal",
 		},
 		{
-			name: "Dark Vertical when mode is dark and orientation is vertical",
-			variations: MediaAssetVariations{
-				LightHorizontal: "light-horizontal",
-				LightVeritical:  "light-vertical",
-				DarkHorizontal:  "dark-horizontal",
-				DarkVeritical:   "dark-vertical",
-			},
 			mode:        "dark",
-			orientation: "vertical",
-			expected:    "dark-vertical",
+			orientation: "",
+			expected:    "dark-horizontal",
 		},
 		{
-			name: "Fallback to Light Horizontal if no matches",
-			variations: MediaAssetVariations{
-				LightHorizontal: "light-horizontal",
-				LightVeritical:  "",
-				DarkHorizontal:  "",
-				DarkVeritical:   "",
-			},
 			mode:        "dark",
 			orientation: "vertical",
-			expected:    "light-horizontal",
-		},
-		{
-			name: "Empty variations fallback to empty string",
-			variations: MediaAssetVariations{
-				LightHorizontal: "",
-				LightVeritical:  "",
-				DarkHorizontal:  "",
-				DarkVeritical:   "",
-			},
-			mode:        "",
-			orientation: "",
-			expected:    "",
+			expected:    "dark-vertical",
 		},
 	}
 
 	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			result := tt.variations.PickVariation(tt.mode, tt.orientation)
+		name := fmt.Sprintf("mode=%s, orientation=%s", tt.mode, tt.orientation)
```

Comment:
If `mode` or `orientation` == "", then these test names might be a bit confusing. 
Should we do 
```suggestion
                 mode := "none"
                 if len(tt.mode) > 0 {
                       mode = tt.mode
                 }
                  orientation := "none"
                 if len(tt.orientation) > 0 {
                       orientation = tt.orientation
                 }
		name := fmt.Sprintf("mode=%s, orientation=%s", mode, orientation)
```

Reply:
This is a nit, so feel free to ignore!